[FIX] account_peppol: fix settings text, search

aliyatastemirova opened the pull request:
The text should be updated since we do not review applications anymore. Also, currently, if a user types a random text in the settings' search bar, the Peppol setting appears. As with all other settings, Peppol settings should be enclosed in `block` tags.

task-3525510

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/odoo/pull/136772).

aliyatastemirova commented:
fyi @tsb-odoo @Olivier-olma @jcs-odoo

smetl commented:
@aliyatastemirova have you changed something else than the adding of the block ?

aliyatastemirova commented:
@smetl 
I changed the xpath and the div to these lines below and removed the title in the h2 tag. Also, I removed some classes here and there.
```
<block id="analytic" position="after">
      <block title="PEPPOL Electronic Document Invoicing">
```
Of course, also adjusted the text about Peppol registration

smetl commented on addons/account_peppol/views/res_config_settings_views.xml:
```diff
@@ -5,200 +5,198 @@
+                                        <label for="is_account_peppol_participant" string='Use PEPPOL Invoicing'/>
+                                        <span class="fa fa-lg fa-building-o" title="Values set here are company-specific."/>
+                                        <div class="text-muted">
+                                            Start sending and receiving documents via Peppol as soon as your registration is complete.
```
Since the wording has changed in stable version, you need to update the .pot file accordingly

smetl commented:
@robodoo r+
